app/Providers/AppServiceProvider.php

```diff
@@ -47,6 +50,14 @@ public function boot(): void
             return $dataService->getOAuth2LoginHelper();
         });
 
+        $this->app->bind(\App\Actions\WordPress\AuthorizeEquipmentAction::class, function($app) {
+            return new AuthorizeEquipmentAction($app->make(WooCommerceApi::class));
+        });
+
+        $this->app->bind(\App\Actions\WordPress\BatchEquipmentAuthorizationAction::class, function($app) {
+            return new BatchAuthorizeEquipmentAction($app->make(AuthorizeEquipmentAction::class));
+        });
+
```

Comment:
You don't need to bind these at all, they will automatically inject the correct resources.